packages/governance/src/ballotBuilder.js

```diff
@@ -0,0 +1,77 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+
+// CHOOSE_N: voter indicates up to N they find acceptable (N might be 1).
+// ORDER: voter lists their choices from most to least favorite.
+// WEIGHT: voter lists their choices, each with a numerical weight. High
+//   numbers are most preferred.
+
+/**
+ * @type {{
+ *   CHOOSE_N: 'choose_n',
+ *   ORDER: 'order',
+ *   WEIGHT: 'weight',
+ * }}
+ */
+const ChoiceMethod = {
+  CHOOSE_N: 'choose_n',
+  ORDER: 'order',
+  WEIGHT: 'weight',
+};
+
+const buildEqualWeightBallot = (
+  method,
+  question,
+  positions,
+  maxChoices = 0,
+) => {
+  const choose = chosenPositions => {
+    assert(
+      chosenPositions.length <= maxChoices,
+      X`only ${maxChoices} position(s) allowed`,
+    );
+
+    for (const position of chosenPositions) {
```

Comment:
This might be better with a single assert of a compound check; e.g., 

```
assert(chosenPositions.every(
          p => positions.includes(p), 
          details`Some positions in ${chosenPositions} are not valid in ${positions}`)
```